reader/components/pdf-recommender/PDFRecommender.jsx

```diff
@@ -1,10 +1,38 @@
-import React, { useEffect, useState } from 'react'
+import React, { useEffect, useState, useRef } from 'react'
 import load from 'little-loader'
-
 import './PDFRecommender.scss'
+import { useGlobalStore } from '../../store'
 
 const PDFRecommender = ({ containerWidth }) => {
+  const [{ ui }] = useGlobalStore()
   const [recommenderLoaded, setRecommenderLoaded] = useState(false)
+  const recommenderRef = useRef()
+
+  useEffect(() => {
+    if (ui.isRelatedPapersScrolled) {
+      const offsetTop =
+        recommenderRef.current.getBoundingClientRect().top +
+        recommenderRef.current.parentNode.scrollTop
+
+      recommenderRef.current.parentNode.scroll({
+        left: 0,
+        top: offsetTop - 1000,
+        behavior: 'auto',
+      })
+
+      // needs to be within timeout otherwise events
+      // get merged and everything is scrolled smoothly not only last 900px
+      setTimeout(
+        () =>
+          recommenderRef.current.parentNode.scroll({
+            left: 0,
+            top: offsetTop - 200,
+            behavior: 'smooth',
+          }),
+        50
+      )
+    }
+  }, [ui.isRelatedPapersScrolled])
```

Comment:
Seems to be really interesting 🙂

Have you tested this on small papers and small windows? (lower than 1000 pixels, and lower than 200 pixels)

Reply:
Not really. Will do it. Thanks

Reply:
It works just fine at least what I tested.

Reply:
Perhaps API works around negative numbers somehow in a good way. It is expectable if think a little bit about but we had to test it.

Thank you!

Reply:
Btw. I am working on some basic Cypress tests. I will create PR once Github actions are released.